### ISSUE TITLE
Publish an app to Rinkeby is not working

### ISSUE THREAD

**Issue by 0xGabi** [aragon/aragon-cli#1107]
## 🐛 Bug Report

I'm getting the following output:

```
aragon apm publish major CounterApp --no-build --env rinkeby
  ✔ Applying version bump (major)
  ✔ Deploy contract
  ✔ Determine contract address for version
  ✔ Prepare files for publishing
  ✔ Generate application artifact
  ✖ Publish intent
    → Only HTTP(S) protocols are supported
✖ Only HTTP(S) protocols are supported
```

I suspect is about the version of aragonPM that we are using. There were an update to use the latest version of `ipfs-http-client` and might be something wrong on apm. We should probably pin to the previous version to fix this.

**Comment by 0xGabi** [comment]
```
Publish test6969.open.aragonpm.eth
    ⠸ Executing newRepoWithVersion on 0x2DF31C5fcBB94656fc98e2953cF0DA03e2885Cfa
      → Generating transaction
(node:19242) UnhandledPromiseRejectionWarning: Error: CONNECTION ERROR: Couldn't connect to node on WS.
    at Object.InvalidConnection (/Users/gabi/code/Aragon/CLI/aragon-cli/packages/cli/node_modules/web3/node_modules/web3-core-helpers/src/errors.js:35:16)
    at WebsocketProvider._timeout (/Users/gabi/code/Aragon/CLI/aragon-cli/packages/cli/node_modules/web3/node_modules/web3-providers-ws/src/index.js:255:48)
    at W3CWebSocket.connection.onclose (/Users/gabi/code/Aragon/CLI/aragon-cli/packages/cli/node_modules/web3/node_modules/web3-providers-ws/src/index.js:155:15)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/gabi/code/Aragon/CLI/aragon-cli/packages/cli/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onClose (/Users/gabi/code/Aragon/CLI/aragon-cli/packages/cli/node_modules/@web3-js/websocket/lib/W3CWebSocket.js:228:10)
    at WebSocketConnection.<anonymous> (/Users/gabi/code/Aragon/CLI/aragon-cli/packages/cli/node_modules/@web3-js/websocket/lib/W3CWebSocket.js:201:17)
    at WebSocketConnection.emit (events.js:193:13)
    at WebSocketConnection.drop (/Users/gabi/code/Aragon/CLI/aragon-cli/packages/cli/node_modules/@web3-js/websocket/lib/WebSocketConnection.js:475:14)
    at /Users/gabi/code/Aragon/CLI/aragon-cli/packages/cli/node_modules/@web3-js/websocket/lib/WebSocketConnection.js:303:18
    at processTicksAndRejections (internal/process/task_queues.js:79:9)
(node:19242) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejectio  ❯ Publish test6969.open.aragonpm.eth
```

**Comment by 0xGabi** [comment]
Fixed with #1111
